WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -422,6 +428,16 @@ - (BOOL)isRowLastRowForSection:(NSIndexPath *)indexPath
     return indexPath.row == (sectionInfo.numberOfObjects - 1);
 }
 
+- (void)trackAppearedIfNeeded
+{
+    if(_trackedViewDisplay) {
+        return;
+    }
+    
+    [WPAnalytics track:WPAnalyticsStatNotificationsAccessed];
+    _trackedViewDisplay = true;
```

Comment:
Since we're refactoring anyway, any particular reason to reference the ivar vs using the accessors?